extensions/amp-accordion/1.0/amp-accordion.js

```diff
@@ -107,16 +112,21 @@ function getState(element, mu) {
       EXPAND_STATE_SHIM_PROP,
       getExpandStateTrigger
     );
-    const props = dict({
+    const sectionProps = dict({
       'key': section,
       'as': sectionShim,
-      'headerAs': headerShim,
-      'contentAs': contentShim,
       'expanded': section.hasAttribute('expanded'),
       'id': section.getAttribute('id'),
       'onExpandStateChange': expandStateShim,
     });
-    return <AccordionSection {...props} />;
+    const headerProps = dict({'as': headerShim});
+    const contentProps = dict({'as': contentShim});
```

Comment:
Add `id`?

Reply:
Sounds good, this will be addressed in follow up PR and is currently tracked in the tracker: #30445

The item is called: `handle protect existing id on content node`